sdcm/cluster.py

```diff
@@ -2766,15 +2769,29 @@ def _parse_cs_results(lines):
         return results
 
     @staticmethod
+    def _parse_gemini_summary_json(json_str):
+
+        results = {'result': {}}
+        try:
+            results = json.loads(json_str)
+
+        except Exception as details:
+            logger.debug("Invalid json document {}".format(details))
```

Comment:
why not return `{}` when handling exception?

Reply:
it will be returned anyway